src/index.ts

```diff
@@ -35,7 +38,11 @@ import TellMeAboutMyTeamScript from './scripts/tell_me_about_my_team.script';
 import TellMeAboutTeamScript from './scripts/tell_me_about_team.script';
 
 function load(robot: RobotWithClient) {
-  robot.client = new Client(robot);
+  loadConfig(robot.logger.error.bind(robot.logger));
+
+  robot.client = new Client(Config.HACKBOT_API_URL.value, Config.HACKBOT_API_PASSWORD.value, robot);

```

Comment:
I love this change; centralising this and adding one place for the 'required'/validation steps makes it cleaner.